src/components/Editor/Runners/HtmlRunner/HtmlRunner.test.js

```diff
@@ -357,3 +367,82 @@ describe("When an allowed link is rendered", () => {
     });
   });
 });
+
+describe("When on desktop", () => {
+  let store;
+
+  beforeEach(() => {
+    setMedia({
+      width: "1000px",
+    });
+    const middlewares = [];
+    const mockStore = configureStore(middlewares);
+    const initialState = {
+      editor: {
+        project: {
+          components: [indexPage],
+        },
+        focussedFileIndices: [0],
+        openFiles: [["index.html"]],
+        codeRunTriggered: true,
+        codeHasBeenRun: true,
+        errorModalShowing: false,
+      },
+    };
+    store = mockStore(initialState);
+    render(
+      <Provider store={store}>
+        <MemoryRouter>
+          <div id="app">
+            <HtmlRunner />
+          </div>
+        </MemoryRouter>
+      </Provider>,
+    );
+  });
+
+  test("There is no run button", () => {
```

Comment:
I always see a run button in the editor - I realise this is me failing to understand something but at what point should the Run button not be available?

Reply:
This isn't really "there is no run button", but there's no run button in the `HtmlRunner`... So on mobile the run button moves to the tab bar in both the code and output panel, and in particular for output that's part of the `PythonRunner`/`HtmlRunner`, whereas on desktop the run button is rendered in `Project` (I think) instead and shows up in the `RunBar` below the code instead.